app/src/components/blocks/deleteDocumentsStage/DeleteDocumentsStage.tsx

```diff
@@ -65,16 +70,35 @@ function DeleteDocumentsStage({
 
         if (fieldValues.deleteDocs === 'yes') {
             setDeletionStage(SUBMISSION_STATE.PENDING);
+            let response;
             try {
-                const response = await deleteAllDocuments({
-                    docType,
-                    nhsNumber: patientNhsNumber,
-                    baseUrl,
-                    baseHeaders,
-                });
-                if (response.status === 200) {
+                if (docType === DOCUMENT_TYPE.LLOYD_GEORGE) {
+                    response = await deleteAllDocuments({
+                        docType,
+                        nhsNumber: patientNhsNumber,
+                        baseUrl,
+                        baseHeaders,
+                    });
+                }
+                if (docType === DOCUMENT_TYPE.ALL) {
+                    response = await deleteAllDocuments({
+                        docType: DOCUMENT_TYPE.ARF,
+                        nhsNumber: patientNhsNumber,
+                        baseUrl,
+                        baseHeaders,
+                    });
+                    response = await deleteAllDocuments({
+                        docType: DOCUMENT_TYPE.LLOYD_GEORGE,
+                        nhsNumber: patientNhsNumber,
+                        baseUrl,
+                        baseHeaders,
```

Comment:
Not entirely sure if this is the implementation we are looking for if you are trying to check the status on multiple requests. There is a more complex but cleaner way of doing this by making an array of promises and reducing their outputs into one status code.

I was also looking for why we are assigning response when we only care about the status. At first it seemed like we were trying to fetch something more than just the status code. We can catchup in around about this block